app/Login.tsx

```diff
@@ -46,13 +45,18 @@ export function Login(props: { onLoginSuccess: () => void }): ReactElement {
             display: "flex",
             flexDirection: "column",
             flexGrow: 4,
-            color: "#FFFFFF",
             fontSize: "12px",
           }}
         >
           <Form.Label>Username</Form.Label>
+          <div style={{ height: "10px" }}></div>
           <Form.Control
-            style={{ backgroundColor: "#B5A8A8" }}
+            style={{
+              width: "160px",
+              height: "32px",
+              flexShrink: "0",
+              background: "#B5A8A8",
```

Comment:
this style is repeated further down, which indicates it should be a separate, reusable component.

Also I'm fairly sure you can pass the numerical values as just numbers, like `width: 160` when you are using the style prop

also also not sure what `flexShrink` is doing here, pls explain

Reply:
replaced Form.Control with StyledFormControl containing styling props
removed flexShrink cos I also don't know what it's doing 
was planning on doing a styling cull in between feature PRs